app/services/support_interface/referee_survey_export.rb

```diff
@@ -17,8 +17,8 @@ def call
           'Experience explanation' => extract_explanation(reference, 'Please rate your experience of giving a reference'),
           'Consent to be contacted' => extract_rating(reference, 'Can we contact you about your experience of giving a reference?'),
           'Contact details' => extract_explanation(reference, 'Can we contact you about your experience of giving a reference?'),
-          'Safe to work with children?' => extract_rating(reference, 'If we asked whether a candidate was safe to work with children, would you feel able to answer?'),
-          'Safe to work with children explanation' => extract_explanation(reference, 'If we asked whether a candidate was safe to work with children, would you feel able to answer?'),
+          'Safe to work with children?' => reference.safeguarding_concerns_status,
+          'Safe to work with children explanation' => reference.safeguarding_concerns,
         }
```

Comment:
I don't have much context, but I'm a little confused by this. Why are we adding this information to the export?

Reply:
It doesn't have to change, but we ask a referee whether there are safeguarding issues twice. It is now on a mandatory page with it's own radio button, whereas previously it was simply a question on the questionnaire.

When the new page was added, the question was not removed from the questionnaire.

Reply:
Aha - I think we're conflating two things here. The questionnaire asks "If we asked whether a candidate was safe to work with children, would you feel able to answer?". We now have a question in the form, which is "Do you know of any reason why #{@application.full_name} should not work with children?". Those are not the same. Does that make sense?

Now that's cleared up, I don't entirely understand why https://sentry.io/organizations/dfe-bat/issues/1721431101 happened. It seems to occur when the `If we asked whether a candidate was safe to work with children, would you feel able to answer?` question is `nil` in the JSON object. Any idea why that could have happened?

Reply:
My understanding is that it was refactored here but the survey export was not updated: https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/2197

Reply:
This makes sense to me. The only bit I'm unclear about is how the boolean values got into the questionnaire data?